packages/cli/src/lib/snapshot/snapshotCommon.ts

```diff
@@ -76,6 +77,23 @@ export async function getTargetOrg(config: Config, target?: string) {
   return cfg.organization;
 }
 
+export function handleSnapshotError(err?: Error) {
```

Comment:
lgtm, just add some unit test for this utils ;)

Reply:
It's kinda tricky to test this use case in the UT. I planned to cover that use case in the E2E tests